src/constants/links.ts

```diff
@@ -1,2 +1,4 @@
 export const defaultProfilePhoto =
 	'https://mod.go.ke/wp-content/uploads/2021/04/default-profile-pic.png';
+
+export const REACT_APP_GOOGLE_REDIRECT = 'http://localhost:3000/auth/redirect';
```

Comment:
you will need manualy change localhost:3000 for each env. for example on staging google will not work now,  it will generate git changes and conflicts